azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/manager/UpdatableOptionsManager.kt

```diff
@@ -24,6 +27,11 @@ internal class UpdatableOptionsManager(
             setSubtitleChangedEventHandler {
                 store.dispatch(CallScreenInfoHeaderAction.UpdateSubtitle(it))
             }
+            /* <CALL_START_TIME> */
+            setShowCallDurationChangedEventHandler {
```

Comment:
Since this event change is not updatable during the call is in progress or after the library instance is created, should this action dispatch be here? Should we place it in a proper place otherwise it might create a confusion to others looking at the code thinking this is an updatable field? 
Or is it ? My understanding for this feature was the timer wouldn't be updatable during call or after instance creation.